### PR TITLE
More explanation in the centaur docs

### PR DESCRIPTION
**Pull request by rhpvorderman**
While working on #4815  I had some issues getting the testing going with centaur. I have updated the documentation so it will be (hopefully) a bit easier or the next person who reads it.

There were some things that I did find in the code and in the centaur tests that I could not find information on. These are marked with `TODO`.  I hope the cromwell devs will fill in the blanks.

### CODE REVIEW

File: docs/developers/Centaur.md [hunk 1]
```diff
@@ -2,7 +2,12 @@ Centaur is an integration testing suite for the [Cromwell](http://github.com/bro
 
 ## Prerequisites
 
-Centaur expects to find a Cromwell server properly configured and running in server mode, listening on port 8000.  This can be configured by modifying the `cromwellUrl` parameter in `application.conf`.
+Centaur expects to find a Cromwell server properly configured and running in server mode, listening on port 8000.  
+This can be configured by modifying the `cromwellUrl` parameter in `application.conf`.
+
+You can get a build of your current cromwell code with [these instructions](Building.md).
+The server can be run with `java -jar <Cromwell JAR> server`.
```
Comment (mcovarr) [inline docs/developers/Centaur.md]: Perhaps a general instruction to run the Cromwell server and a link to the existing documentation in `../CommandLine.md`?

File: docs/developers/Centaur.md [hunk 2]
```diff
@@ -74,5 +90,15 @@ The `testFormat` field can be one of the following, case insensitive:
...
+This will save the metadata in `metadata.json`.
+
+For any metadata values or outputExpectations which require workflow ID (i.e, file paths), use `<<UUID>>` as a placeholder instead. For example:
+* `"calls.hello.hello.stdout": "gs://google-project/jes/root/wdl/<<UUID>>/call-task/task-stdout.log"`
+
+In case the absolute path the cromwell root is used (for example: `/home/my_user/projects/cromwell/cromwell-executions`)
+ you can use `<<WORKFLOW_ROOT>>` as a replacement. 
+* `"calls.hello.hello.exit_code": "<<WORKFLOW_ROOT>>/call-hello/execution/exit_code"`
+
+TODO: `<<CACHE_HIT_UUID>>` In what sort of tests can you use this?
```
Comment (mcovarr) [inline docs/developers/Centaur.md]: Call caching tests! 😄 These magic values should expand to the cache hit workflow's ID.

Comment (cjllanwarne) [inline docs/developers/Centaur.md]: Used if the testFormat is `runtwiceexpectingcallcaching`, for example, and filled in with the ID of the first time the workflow runs

**Comment by cjllanwarne** [comment]
thanks @rhpvorderman

**Pull request merged**